kvirt/k3s/bootstrap.sh

```diff
@@ -1,15 +1,10 @@
-{% set extra_args = [] %}
-{% for component in disabled_components %}
-{% do extra_args.append("--disable " + component) %}
-{% endfor %}
-apt-get -y install curl
-{% if masters > 1 %}
-curl -sfL https://get.k3s.io | {{ "INSTALL_K3S_EXEC='--flannel-backend=none'" if sdn != "flannel" else '' }} INSTALL_K3S_CHANNEL={{ install_k3s_channel }} INSTALL_K3S_VERSION={{ install_k3s_version if install_k3s_version != "latest" else '' }} K3S_TOKEN={{ token }} sh -s - server --cluster-init {{ extra_args|join(" ") }}
-export IP={{ api_ip }}
-{% else %}
-curl -sfL https://get.k3s.io | {{ "INSTALL_K3S_EXEC='--flannel-backend=none'" if sdn != "flannel" else '' }} INSTALL_K3S_CHANNEL={{ install_k3s_channel }} INSTALL_K3S_VERSION={{ install_k3s_version if install_k3s_version != "latest" else '' }} K3S_TOKEN={{ token }} sh -s - server {{ extra_args|join(" ") }}
-export IP=$(hostname -I | cut -f1 -d" ")
+{% if extra_master_args %}
+{% set extra_args = extra_master_args %}
 {% endif %}
+
+apt-get -y install curl
```

Comment:
And what ended up being the conclusion to the removal of cURL ?? In regards to that executable being a security liability?

Reply:
i believe it s contradictory to remove the main asset used for installation on this platform and recommended by k3s.
but i can hear your concern and since you re the main consumer of this, i m fine having it removed once install is finished

Reply:
Then again ... you have a fair argument and I could just have a line in a `cmds` clause/section that removes *cURL*. However, is having a `cmds` section supported now if one have workers to install and therefore join a deployed K3s cluster? Remember that we found out - last week - that the cmds one specify in a `cmds` section overwrites the ones in the `workers.yml` file.

See my changes to the KubeVirt/kvirt config.py file > https://github.com/karmab/kcli/pull/284/files#diff-7ae6ac623bc6fbb649f65c73653c64606afbc0b59fb00add7601ef2b762c7cb7 ... I know we talked about this not being the optimal way of handling it. So I'm more pointing to it to ensure that we remember this issue and get it 👍

Reply:
changed the code to remove curl at the end of the execution of each node